crypto/rand/rand_egd.c

```diff
@@ -54,6 +54,69 @@ struct sockaddr_un {
 # include <string.h>
 # include <errno.h>
 
+#if defined(OPENSSL_SYSNAME_TANDEM)
```

Comment:
This needs to be nested

#-if

and then the defines within 
#--define

(where - is a space)